samples/server/petstore/jaxrs-spec/swagger.json

```diff
@@ -760,24 +760,24 @@
           "description" : "None",
           "required" : true,
           "type" : "number",
-          "maximum" : 543.2,
-          "minimum" : 32.1
+          "maximum" : 543.200000000000045474735088646411895751953125,
+          "minimum" : 32.10000000000000142108547152020037174224853515625
         }, {
           "name" : "float",
           "in" : "formData",
           "description" : "None",
           "required" : false,
           "type" : "number",
-          "maximum" : 987.6,
+          "maximum" : 987.6000000000000227373675443232059478759765625,
```

Comment:
I did a test with this PR and I got different results when running ./bin/jaxrs-spec-petstore-server.sh: https://github.com/swagger-api/swagger-codegen/commit/faa453d7f9b51b78ccc1e6c55a73d1aa5014024a, which has new issues (e.g. integer becomes decimal...)